.github/workflows/testing.yml

```diff
@@ -59,45 +59,60 @@ jobs:
 
   functional: #----------------------------------------------------------------------
     name: Functional - WP ${{ matrix.wp }} on PHP ${{ matrix.php }} with MySQL ${{ matrix.mysql }}
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
       fail-fast: false
       matrix:
-        php: ['5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0']
+        php: ['5.6', '7.0', '7.1', '7.2', '7.3', '7.4']
```

Comment:
Note the removal does not stop PHP8 running, but to remove combinations, and continue using exclude, this seemed necessary.